content/source/layouts/flexibleengine.erb

```diff
@@ -0,0 +1 @@
+ext/providers/flexibleengine/website/flexibleengine.erb
```

Comment:
I think this should be "../../../ext/providers/flexibleengine/website/flexibleengine.erb"

Reply:
Found those, thanks!